tests/test_flask_pyoidc.py

```diff
@@ -674,3 +676,221 @@ def test_should_return_None_if_token_refresh_request_fails(self):
             session.update(access_token=access_token, expires_in=-10, refresh_token='refresh-token')
             assert authn.valid_access_token(force_refresh=True) is None
             assert session.access_token == access_token
+
+    def test_should_check_for_authorization_header(self):
+
+        authn = self.init_app()
+        with self.app.test_request_context('/'):
+            self.assertFalse(authn._check_authorization_header(flask.request))
+            flask.request.headers = {
+                'Authorization': 'Bearer access_token'
+            }
+            self.assertTrue(authn._check_authorization_header(flask.request))
```

Comment:
This should be `assert authn._check_authorization_header(flask.request) == True` to match existing style (which doesn't use the unittest assert-helpers).
(This applies to all the added tests.)

Reply:
As `assert` evaluates to True or False, ` assert authn._check_authorization_header(flask.request)` is enough and complies with PEP8 convention.